WhatAndroid/src/main/java/what/whatandroid/login/LoggedInActivity.java

```diff
@@ -62,34 +55,7 @@
 	private Login loginTask;
 	private LogoutTask logoutTask;
 
-	/**
-	 * Initialize MySoup so that we can start making API requests
-	 */
-	public static void initSoup(){
```

Comment:
Why was this removed? The way the WhatAPI works is that we need to set some static information like the site name, if we're using SSL and our user agent. Without this we won't be able to make API requests to the site. Related to this the site string shouldn't have been removed either.

I haven't had time to fully go through the PR and try testing it but this stuck out.